[BUG] Install deps and build on macOS failed.

**Describe the bug**
install deps on macOS failed.

```shell
[ 50%] Linking CXX shared library ../../shared-lib/libvineyard_malloc.dylib
[ 50%] Built target vineyard_malloc
[ 50%] Building CXX object modules/migrate/CMakeFiles/vineyard-migrate.dir/vineyard_migrate.cc.o
[ 51%] Building CXX object modules/basic/CMakeFiles/vineyard_basic.dir/ds/tensor.cc.o
1 warning generated.
[ 52%] Building CXX object CMakeFiles/_C.dir/python/error.cc.o
[ 52%] Building CXX object CMakeFiles/_C.dir/python/pybind11_utils.cc.o
1 warning generated.
[ 52%] Building CXX object modules/migrate/CMakeFiles/vineyard-migrate-stream.dir/vineyard_migrate_stream.cc.o
1 warning generated.
[ 52%] Building CXX object modules/basic/CMakeFiles/vineyard_basic.dir/ds/types.cc.o
[ 53%] Building CXX object modules/basic/CMakeFiles/vineyard_basic.dir/stream/byte_stream.cc.o
[ 53%] Building CXX object CMakeFiles/_C.dir/python/vineyard.cc.o
[ 54%] Linking CXX executable ../../bin/vineyard-migrate
[ 54%] Built target vineyard-migrate
[ 54%] Building CXX object modules/basic/CMakeFiles/vineyard_basic.dir/stream/dataframe_stream.cc.o
[ 56%] Building CXX object modules/basic/CMakeFiles/vineyard_basic.dir/stream/parallel_stream.cc.o
[ 56%] Building CXX object modules/basic/CMakeFiles/vineyard_basic.dir/stream/recordbatch_stream.cc.o
[ 57%] Linking CXX shared module shared-lib/_C.cpython-39-darwin.so
[ 58%] Linking CXX executable ../../bin/vineyard-migrate-stream
[ 58%] Built target vineyard-migrate-stream
[ 58%] Copying the migration artifacts to the python package.
[ 58%] Built target vineyard_migrate_python
/opt/homebrew/opt/llvm/bin/llvm-strip: error: unsupported load command (cmd=0x80000034)
make[2]: *** [shared-lib/_C.cpython-39-darwin.so] Error 1
make[2]: *** Deleting file `shared-lib/_C.cpython-39-darwin.so'
make[1]: *** [CMakeFiles/_C.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
1 warning generated.
[ 58%] Linking CXX shared library ../../shared-lib/libvineyard_basic.dylib
[ 58%] Built target vineyard_basic
1 warning generated.
[ 58%] Linking CXX static library ../Binaries/libcpprest.a
[ 58%] Built target cpprest
make: *** [all] Error 2
```

**To Reproduce**
Steps to reproduce the behavior:
- ./scripts/install_deps.sh --dev

**Environment (please complete the following information):**
 - GraphScope version: master
 - OS: macOS 12.4 with m1 pro
 - Python: 3.9 Arm64

Can you please try to update the `scripts/install_deps.sh` and update the following lines (https://github.com/alibaba/GraphScope/blob/f25fc429f359942201ccd9bec8b304996d76defa/scripts/install_deps.sh#L520) as

```
cmake -DCPPKAFKA_DISABLE_TESTS=ON  -DCPPKAFKA_DISABLE_EXAMPLES=ON .. && make -j$(nproc)
```

> > 
> 
> I meet another issue. In `scripts/install_deps.sh`, shell function `install_cppkafka` does not determine the homebrew_prefix, after fix that, I found that cppkafka can not compile.
> 
> ```shell
> -- Build files have been written to: /tmp/cppkafka/build
> [  2%] Building CXX object src/CMakeFiles/cppkafka.dir/configuration.cpp.o
> [  8%] Building CXX object src/CMakeFiles/cppkafka.dir/topic_configuration.cpp.o
> [  8%] Building CXX object src/CMakeFiles/cppkafka.dir/configuration_option.cpp.o
> [ 11%] Building CXX object src/CMakeFiles/cppkafka.dir/topic.cpp.o
> [ 13%] Building CXX object src/CMakeFiles/cppkafka.dir/message_timestamp.cpp.o
> [ 16%] Building CXX object src/CMakeFiles/cppkafka.dir/queue.cpp.o
> [ 19%] Building CXX object src/CMakeFiles/cppkafka.dir/message.cpp.o
> [ 22%] Building CXX object src/CMakeFiles/cppkafka.dir/buffer.cpp.o
> [ 27%] Building CXX object src/CMakeFiles/cppkafka.dir/message_internal.cpp.o
> [ 27%] Building CXX object src/CMakeFiles/cppkafka.dir/exceptions.cpp.o
> [ 30%] Building CXX object src/CMakeFiles/cppkafka.dir/topic_partition.cpp.o
> [ 33%] Building CXX object src/CMakeFiles/cppkafka.dir/topic_partition_list.cpp.o
> [ 36%] Building CXX object src/CMakeFiles/cppkafka.dir/metadata.cpp.o
> [ 38%] Building CXX object src/CMakeFiles/cppkafka.dir/group_information.cpp.o
> [ 41%] Building CXX object src/CMakeFiles/cppkafka.dir/error.cpp.o
> [ 44%] Building CXX object src/CMakeFiles/cppkafka.dir/event.cpp.o
> [ 47%] Building CXX object src/CMakeFiles/cppkafka.dir/kafka_handle_base.cpp.o
> [ 50%] Building CXX object src/CMakeFiles/cppkafka.dir/producer.cpp.o
> [ 52%] Building CXX object src/CMakeFiles/cppkafka.dir/consumer.cpp.o
> [ 55%] Building CXX object src/CMakeFiles/cppkafka.dir/utils/backoff_performer.cpp.o
> [ 58%] Building CXX object src/CMakeFiles/cppkafka.dir/utils/backoff_committer.cpp.o
> [ 61%] Building CXX object src/CMakeFiles/cppkafka.dir/utils/poll_strategy_base.cpp.o
> [ 63%] Building CXX object src/CMakeFiles/cppkafka.dir/utils/roundrobin_poll_strategy.cpp.o
> /tmp/cppkafka/src/producer.cpp:148:54: warning: object backing the pointer will be destroyed at the end of the full-expression [-Wdangling-gsl]
>                                     RD_KAFKA_V_TOPIC(message.get_topic().data()),
>                                                      ^~~~~~~~~~~~~~~~~~~
> /opt/homebrew/include/librdkafka/rdkafka.h:1152:60: note: expanded from macro 'RD_KAFKA_V_TOPIC'
>         _LRK_TYPECHECK(RD_KAFKA_VTYPE_TOPIC, const char *, topic),             \
>                                                            ^~~~~
> /opt/homebrew/include/librdkafka/rdkafka.h:115:47: note: expanded from macro '_LRK_TYPECHECK'
>                         TYPE __t RD_UNUSED = (ARG);                            \
>                                               ^~~
> 1 warning generated.
> [ 66%] Linking CXX shared library lib/libcppkafka.dylib
> [ 66%] Built target cppkafka
> [ 69%] Building CXX object tests/CMakeFiles/cppkafka_tests.dir/configuration_test.cpp.o
> [ 77%] Building CXX object tests/CMakeFiles/cppkafka_tests.dir/kafka_handle_base_test.cpp.o
> [ 77%] Building CXX object tests/CMakeFiles/cppkafka_tests.dir/topic_partition_list_test.cpp.o
> [ 77%] Building CXX object tests/CMakeFiles/cppkafka_tests.dir/compacted_topic_processor_test.cpp.o
> [ 86%] Building CXX object tests/CMakeFiles/cppkafka_tests.dir/roundrobin_poll_test.cpp.o
> [ 86%] Building CXX object tests/CMakeFiles/cppkafka_tests.dir/consumer_test.cpp.o
> [ 86%] Building CXX object tests/CMakeFiles/cppkafka_tests.dir/buffer_test.cpp.o
> [ 88%] Building CXX object tests/CMakeFiles/cppkafka_tests.dir/test_utils.cpp.o
> [ 91%] Building CXX object tests/CMakeFiles/cppkafka_tests.dir/headers_test.cpp.o
> [ 94%] Building CXX object tests/CMakeFiles/cppkafka_tests.dir/producer_test.cpp.o
> [ 97%] Building CXX object tests/CMakeFiles/cppkafka_tests.dir/test_main.cpp.o
> In file included from /tmp/cppkafka/tests/test_main.cpp:3:
> /tmp/cppkafka/third_party/Catch2/single_include/catch.hpp:5025:13: error: unrecognized instruction mnemonic, did you mean: bit, cnt, hint, ins, not?
>             CATCH_BREAK_INTO_DEBUGGER();
>             ^
> /tmp/cppkafka/third_party/Catch2/single_include/catch.hpp:4747:75: note: expanded from macro 'CATCH_BREAK_INTO_DEBUGGER'
>     #define CATCH_BREAK_INTO_DEBUGGER() if( Catch::isDebuggerActive() ) { CATCH_TRAP(); }
>                                                                           ^
> /tmp/cppkafka/third_party/Catch2/single_include/catch.hpp:4726:34: note: expanded from macro 'CATCH_TRAP'
>     #define CATCH_TRAP() __asm__("int $3\n" : : ) /* NOLINT */
>                                  ^
> <inline asm>:1:2: note: instantiated into assembly here
>         int $3
>         ^
> 1 error generated.
> make[2]: *** [tests/CMakeFiles/cppkafka_tests.dir/test_main.cpp.o] Error 1
> make[1]: *** [tests/CMakeFiles/cppkafka_tests.dir/all] Error 2
> make: *** [all] Error 2
> ```

Can you please try to update the `scripts/install_deps.sh` and update the following lines (https://github.com/alibaba/GraphScope/blob/f25fc429f359942201ccd9bec8b304996d76defa/scripts/install_deps.sh#L520) as

```
cmake -DCPPKAFKA_DISABLE_TESTS=ON  -DCPPKAFKA_DISABLE_EXAMPLES=ON .. && make -j$(nproc)
```

> Can you please try to update the `scripts/install_deps.sh` and update the following lines (
> 
> https://github.com/alibaba/GraphScope/blob/f25fc429f359942201ccd9bec8b304996d76defa/scripts/install_deps.sh#L520
> 
> ) as
> ```
> cmake -DCPPKAFKA_DISABLE_TESTS=ON  -DCPPKAFKA_DISABLE_EXAMPLES=ON .. && make -j$(nproc)
> ```
> 
> > > 
> > 
> > 
> > I meet another issue. In `scripts/install_deps.sh`, shell function `install_cppkafka` does not determine the homebrew_prefix, after fix that, I found that cppkafka can not compile.
> > ```shell
> > -- Build files have been written to: /tmp/cppkafka/build
> > [  2%] Building CXX object src/CMakeFiles/cppkafka.dir/configuration.cpp.o
> > [  8%] Building CXX object src/CMakeFiles/cppkafka.dir/topic_configuration.cpp.o
> > [  8%] Building CXX object src/CMakeFiles/cppkafka.dir/configuration_option.cpp.o
> > [ 11%] Building CXX object src/CMakeFiles/cppkafka.dir/topic.cpp.o
> > [ 13%] Building CXX object src/CMakeFiles/cppkafka.dir/message_timestamp.cpp.o
> > [ 16%] Building CXX object src/CMakeFiles/cppkafka.dir/queue.cpp.o
> > [ 19%] Building CXX object src/CMakeFiles/cppkafka.dir/message.cpp.o
> > [ 22%] Building CXX object src/CMakeFiles/cppkafka.dir/buffer.cpp.o
> > [ 27%] Building CXX object src/CMakeFiles/cppkafka.dir/message_internal.cpp.o
> > [ 27%] Building CXX object src/CMakeFiles/cppkafka.dir/exceptions.cpp.o
> > [ 30%] Building CXX object src/CMakeFiles/cppkafka.dir/topic_partition.cpp.o
> > [ 33%] Building CXX object src/CMakeFiles/cppkafka.dir/topic_partition_list.cpp.o
> > [ 36%] Building CXX object src/CMakeFiles/cppkafka.dir/metadata.cpp.o
> > [ 38%] Building CXX object src/CMakeFiles/cppkafka.dir/group_information.cpp.o
> > [ 41%] Building CXX object src/CMakeFiles/cppkafka.dir/error.cpp.o
> > [ 44%] Building CXX object src/CMakeFiles/cppkafka.dir/event.cpp.o
> > [ 47%] Building CXX object src/CMakeFiles/cppkafka.dir/kafka_handle_base.cpp.o
> > [ 50%] Building CXX object src/CMakeFiles/cppkafka.dir/producer.cpp.o
> > [ 52%] Building CXX object src/CMakeFiles/cppkafka.dir/consumer.cpp.o
> > [ 55%] Building CXX object src/CMakeFiles/cppkafka.dir/utils/backoff_performer.cpp.o
> > [ 58%] Building CXX object src/CMakeFiles/cppkafka.dir/utils/backoff_committer.cpp.o
> > [ 61%] Building CXX object src/CMakeFiles/cppkafka.dir/utils/poll_strategy_base.cpp.o
> > [ 63%] Building CXX object src/CMakeFiles/cppkafka.dir/utils/roundrobin_poll_strategy.cpp.o
> > /tmp/cppkafka/src/producer.cpp:148:54: warning: object backing the pointer will be destroyed at the end of the full-expression [-Wdangling-gsl]
> >                                     RD_KAFKA_V_TOPIC(message.get_topic().data()),
> >                                                      ^~~~~~~~~~~~~~~~~~~
> > /opt/homebrew/include/librdkafka/rdkafka.h:1152:60: note: expanded from macro 'RD_KAFKA_V_TOPIC'
> >         _LRK_TYPECHECK(RD_KAFKA_VTYPE_TOPIC, const char *, topic),             \
> >                                                            ^~~~~
> > /opt/homebrew/include/librdkafka/rdkafka.h:115:47: note: expanded from macro '_LRK_TYPECHECK'
> >                         TYPE __t RD_UNUSED = (ARG);                            \
> >                                               ^~~
> > 1 warning generated.
> > [ 66%] Linking CXX shared library lib/libcppkafka.dylib
> > [ 66%] Built target cppkafka
> > [ 69%] Building CXX object tests/CMakeFiles/cppkafka_tests.dir/configuration_test.cpp.o
> > [ 77%] Building CXX object tests/CMakeFiles/cppkafka_tests.dir/kafka_handle_base_test.cpp.o
> > [ 77%] Building CXX object tests/CMakeFiles/cppkafka_tests.dir/topic_partition_list_test.cpp.o
> > [ 77%] Building CXX object tests/CMakeFiles/cppkafka_tests.dir/compacted_topic_processor_test.cpp.o
> > [ 86%] Building CXX object tests/CMakeFiles/cppkafka_tests.dir/roundrobin_poll_test.cpp.o
> > [ 86%] Building CXX object tests/CMakeFiles/cppkafka_tests.dir/consumer_test.cpp.o
> > [ 86%] Building CXX object tests/CMakeFiles/cppkafka_tests.dir/buffer_test.cpp.o
> > [ 88%] Building CXX object tests/CMakeFiles/cppkafka_tests.dir/test_utils.cpp.o
> > [ 91%] Building CXX object tests/CMakeFiles/cppkafka_tests.dir/headers_test.cpp.o
> > [ 94%] Building CXX object tests/CMakeFiles/cppkafka_tests.dir/producer_test.cpp.o
> > [ 97%] Building CXX object tests/CMakeFiles/cppkafka_tests.dir/test_main.cpp.o
> > In file included from /tmp/cppkafka/tests/test_main.cpp:3:
> > /tmp/cppkafka/third_party/Catch2/single_include/catch.hpp:5025:13: error: unrecognized instruction mnemonic, did you mean: bit, cnt, hint, ins, not?
> >             CATCH_BREAK_INTO_DEBUGGER();
> >             ^
> > /tmp/cppkafka/third_party/Catch2/single_include/catch.hpp:4747:75: note: expanded from macro 'CATCH_BREAK_INTO_DEBUGGER'
> >     #define CATCH_BREAK_INTO_DEBUGGER() if( Catch::isDebuggerActive() ) { CATCH_TRAP(); }
> >                                                                           ^
> > /tmp/cppkafka/third_party/Catch2/single_include/catch.hpp:4726:34: note: expanded from macro 'CATCH_TRAP'
> >     #define CATCH_TRAP() __asm__("int $3\n" : : ) /* NOLINT */
> >                                  ^
> > <inline asm>:1:2: note: instantiated into assembly here
> >         int $3
> >         ^
> > 1 error generated.
> > make[2]: *** [tests/CMakeFiles/cppkafka_tests.dir/test_main.cpp.o] Error 1
> > make[1]: *** [tests/CMakeFiles/cppkafka_tests.dir/all] Error 2
> > make: *** [all] Error 2
> > ```
> 
> Can you please try to update the `scripts/install_deps.sh` and update the following lines (
> 
> https://github.com/alibaba/GraphScope/blob/f25fc429f359942201ccd9bec8b304996d76defa/scripts/install_deps.sh#L520
> 
> ) as
> ```
> cmake -DCPPKAFKA_DISABLE_TESTS=ON  -DCPPKAFKA_DISABLE_EXAMPLES=ON .. && make -j$(nproc)
> ```

I have already tried this myself. The deps can be installed successfully, but compile while build gae.
```shell
[ 64%] Linking CXX shared library libgs_proto.dylib
[ 64%] Built target gs_proto
[ 67%] Linking CXX shared library libgs_util.dylib
[ 67%] Built target gs_util
[ 71%] Building CXX object CMakeFiles/grape_engine.dir/core/grape_engine.cc.o
[ 78%] Building CXX object CMakeFiles/grape_engine.dir/core/grape_instance.cc.o
[ 78%] Building CXX object CMakeFiles/grape_engine.dir/core/java/javasdk.cc.o
[ 82%] Building CXX object CMakeFiles/grape_engine.dir/core/object/dynamic.cc.o
[ 85%] Building CXX object CMakeFiles/grape_engine.dir/core/server/command_detail.cc.o
[ 89%] Building CXX object CMakeFiles/grape_engine.dir/core/server/dispatcher.cc.o
[ 92%] Building CXX object CMakeFiles/grape_engine.dir/core/server/graphscope_service.cc.o
[ 96%] Building CXX object CMakeFiles/grape_engine.dir/core/server/analytical_server.cc.o
/Users/waruto/gitrepos/GraphScope/analytical_engine/core/grape_engine.cc:184:46: error: reference to type 'const google::LogMessageTime' could not bind to an lvalue of type 'const struct ::tm *'
                                             tm_time, message, message_len)
                                             ^~~~~~~
/opt/homebrew/include/glog/logging.h:1777:53: note: passing argument to parameter 'logmsgtime' here
                              const LogMessageTime &logmsgtime,
                                                    ^
/Users/waruto/gitrepos/GraphScope/analytical_engine/core/grape_engine.cc:188:46: error: reference to type 'const google::LogMessageTime' could not bind to an lvalue of type 'const struct ::tm *'
                                             tm_time, message, message_len)
                                             ^~~~~~~
/opt/homebrew/include/glog/logging.h:1777:53: note: passing argument to parameter 'logmsgtime' here
                              const LogMessageTime &logmsgtime,
                                                    ^
2 errors generated.
make[3]: *** [CMakeFiles/grape_engine.dir/core/grape_engine.cc.o] Error 1
make[3]: *** Waiting for unfinished jobs....
make[2]: *** [CMakeFiles/grape_engine.dir/all] Error 2
make[1]: *** [all] Error 2
make: *** [gae] Error 2
```

--- a/scripts/install_deps.sh
+++ b/scripts/install_deps.sh
@@ -14,8 +14,8 @@ readonly GREEN="\033[0;32m"
 readonly NC="\033[0m" # No Color
 
 readonly GRAPE_BRANCH="master" # libgrape-lite branch
-readonly V6D_VERSION="0.5.0"  # vineyard version
-readonly V6D_BRANCH="v0.5.0" # vineyard branch
+readonly V6D_VERSION="0.5.2"  # vineyard version
+readonly V6D_BRANCH="v0.5.2" # vineyard branch
 
 readonly OUTPUT_ENV_FILE="${HOME}/.graphscope_env"
 IS_IN_WSL=false && [[ ! -z "${IS_WSL}" || ! -z "${WSL_DISTRO_NAME}" ]] && IS_IN_WSL=true
@@ -385,12 +385,7 @@ write_envs_config() {
   fi
 
   if [[ "${PLATFORM}" == *"Darwin"* ]]; then
-    if [[ "$(uname -m)" == *"x86_64"* ]]; then
-      declare -r homebrew_prefix=/usr/local
-    else
-      # Apple Silicon: packages are installed under /opt/homebrew by default
-      declare -r homebrew_prefix=/opt/homebrew
-    fi
+    declare -r homebrew_prefix=$(brew --prefix)
     {
       echo "export CC=${homebrew_prefix}/opt/llvm/bin/clang"
       echo "export CXX=${homebrew_prefix}/opt/llvm/bin/clang++"
@@ -507,8 +502,9 @@ install_cppkafka() {
   fi
 
   if [[ "${PLATFORM}" == *"Darwin"* ]]; then
-    export LDFLAGS="-L/usr/local/opt/openssl@3/lib"
-    export CPPFLAGS="-I/usr/local/opt/openssl@3/include"
+    declare -r homebrew_prefix=$(brew --prefix)
+    export LDFLAGS="-L${homebrew_prefix}/opt/openssl@3/lib"
+    export CPPFLAGS="-I${homebrew_prefix}/opt/openssl@3/include"
   fi
 
   check_and_remove_dir "/tmp/cppkafka"
@@ -517,7 +513,7 @@ install_cppkafka() {
   pushd /tmp/cppkafka
   git submodule update --init
   mkdir -p build && pushd build
-  cmake .. && make -j$(nproc)
+  cmake -DCPPKAFKA_DISABLE_TESTS=ON  -DCPPKAFKA_DISABLE_EXAMPLES=ON .. && make -j$(nproc)
   sudo make install && popd
   popd
 

